ansible/roles/baselayout/vars/main.yml

```diff
@@ -43,7 +43,7 @@ packages: {
   centos7_x64: ['git2u','centos-release-scl',], # centos-release-scl is required to enable SCLo
                                                # but we do it manually in partials/repo/centos7.yml for arm64
   centos7: [
-    'ccache,gcc-c++,devtoolset-6,sudo',
+    'ccache,gcc-c++,devtoolset-6,sudo,git',
```

Comment:
You mentioned `devtoolset-6-libatomic-devel`

Reply:
While you're here could you add `python3` (or equivalent)?

Reply:
makes sense that both python3 and libatomic should be here.